crates/core/src/util/typed_prometheus.rs

```diff
@@ -122,6 +122,7 @@ impl_prometheusvalue_string!(
     Address,
     TransactionType,
     AbiCall,
+    bool,
```

Comment:
Out of curiosity, suppose in the future we wanted to track something with 3 or more states, which in rust you'd naturally represent as `enum X { A, B, C }`. How we would do that with prometheus?

I'm asking partly just out of curiosity, but also wanting to understand if we'd regret this change when we have a third variant pop up.